modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceShipmentServiceImpl.java

```diff
@@ -139,17 +139,11 @@ public List<CommerceShipment> getCommerceShipments(
 			getPermissionChecker(), null,
```

Comment:
@dantewang do you think this _portletResourcePermission.contains() is used incorrectly?
it returns a boolean that is not used unless it has side effect.

Reply:
@julschong Yes, I think so. Please fix it with a separate ticket and send to Commerce team for review, or report to Commerce team.

Reply:
@dantewang I sent a PR to commerce team here: https://github.com/liferay-commerce/liferay-portal/pull/3484
this PR will likely need to wait